docs/REFERENCE.md

```diff
@@ -376,14 +376,14 @@ The order of prefixes is important, the order is `run_`, `create_`, `modify_`,
 
 Different target types allow different prefixes and suffixes:
 
-| Target type   | Allowed prefixes                                                      | Allowed suffixes |
-| ------------- | --------------------------------------------------------------------- | ---------------- |
-| Directory     | `exact_`, `private_`, `dot_`                                          | *none*           |
-| Regular file  | `encrypted_`, `private_`, `executable_`, `dot_`                       | `.tmpl`          |
-| Create file   | `create_`, `encrypted_`, `private_`, `executable_`, `dot_`            | `.tmpl`          |
-| Modify file   | `modify_`, `encrypted_`, `private_`, `executable_`, `dot_`            | `.tmpl`          |
-| Script        | `run_`, `once_`, `before_` or `after_`                                | `.tmpl`          |
-| Symbolic link | `symlink_`, `dot_`,                                                   | `.tmpl`          |
+| Target type   | Allowed prefixes                                                        | Allowed suffixes |
+| ------------- | ----------------------------------------------------------------------- | ---------------- |
+| Directory     | `exact_`, `private_`, `dot_`                                            | *none*           |
+| Regular file  | `encrypted_`, `private_`, `executable_`, `dot_`                         | `.tmpl`          |
+| Create file   | `create_`, `encrypted_`, `private_`, `executable_`, `dot_`              | `.tmpl`          |
+| Modify file   | `modify_`, `encrypted_`, `private_`, `executable_`, `dot_`              | `.tmpl`          |
+| Script        | `run_` (required), `once_`, `before_` or `after_` (optional, exclusive) | `.tmpl`          |
```

Comment:
`run_` is required for scripts, as `create_`, `modify_`, and `symlink_` are also required for their respective target types. From this change, it's not clear what combination of attributes `(optional, exclusive)` applies to.

I think it's better to explain this in the text rather than squeeze it into the table.